web/styles/dark_theme.css

```diff
@@ -1543,6 +1543,26 @@
             }
         }
     }
+
+    .tabs-container {
+        background-color: hsl(0deg 0% 0% / 30%);
+
+        & i {
+            color: hsl(0deg 0% 80%);
+        }
+
+        .selected-tab {
+            background-color: hsl(0deg 0% 17% / 50%);
+            border: 1px solid hsl(0deg 0% 100% / 12%);
+        }
+
+        .tab-option {
+            &:hover:not(.selected-tab) {
+                background-color: hsl(0deg 0% 17% / 50%);
+                border: 1px solid hsl(0deg 0% 100% / 12%);
+            }
+        }
+    }
```

Comment:
This should be discussed in #design and can be fixed in a follow-up, but playing with this, it feels a bit weird that none of these options are selected for a "default configuration" topic.

Reply:
[CZO Discussion](https://chat.zulip.org/#narrow/stream/101-design/topic/Follow.20Topics.20UI/near/1614625)

Reply:
#26486